src/components/Timeline.js

```diff
@@ -2,9 +2,35 @@ import React from 'react';
 import './Timeline.css';
 import TimelineEvent from './TimelineEvent';
 
-const Timeline = () => {
-  
-  return;
+// For wave 2, you will implement the Timeline component. The Timeline component will take one prop, 
+// events, an array of JavaScript objects. 
+// The render function will use the array to render a set of TimelineEvent components.
+
+// Then read in the /src/data/timeline.json file into an array of objects in /src/App.js. 
+// In the render function of the App component, render a TimeLine component using the array of objects as the events prop.
+
+const Timeline = (props) => {
+
+  const timeEvents = props.events.map((event, i) => {
+    return (
+      <ul>
+        <li key={i}> 
```

Comment:
The "key" should be in the outer layer of the return value from `map`.

```suggestion
      <ul key={i}>
        <li> 
```